libraries/GCS_MAVLink/GCS_FTP.cpp

```diff
@@ -116,7 +116,11 @@ bool GCS_MAVLINK::send_ftp_reply(const pending_ftp &reply)
         reply.chan,
         0, reply.sysid, reply.compid,
         payload);
-    ftp.last_send_ms = AP_HAL::millis();
+    if (reply.req_opcode == FTP_OP::TerminateSession) {
```

Comment:
I think I'm missing it, but what was the code that we were tripping over before? As far as I can tell everything that had a timeout check was all associated with a check that the file descriptor was set, and if we just did a session termination the fd should have been -1.